tests/lib/log/log.go

```diff
@@ -0,0 +1,59 @@
+package log
```

Comment:
Thanks for the idea to use Go's default log, it seems much better now. Enabling transparent file/stdout got pretty cool. I was not able to fix the leak of the tear down function to close the file although... sorry =/. Usage is uniform, but for stdout the teardown does nothing.